### PR TITLE
All Classes and Interfaces were added, resulting Array was generated

### PR DESCRIPTION
**Pull request by Artem-Kushniruk**

### CODE REVIEW

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public Random randomObjectInitialization = new Random();
```
Comment (MaliukDaria) [inline src/main/java/core/basesyntax/ColorSupplier.java:6]:
```suggestion
    public Random random = new Random();
```

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    private final String[] colors = {"green", "yellow", "blue", "white", "black"};
+
+    public String getRandomColor() {
+        int randomNumber = randomObjectInitialization.nextInt(colors.length);
+        return colors[randomNumber];
+    }
+
```
Comment (MaliukDaria) [inline src/main/java/core/basesyntax/ColorSupplier.java:13]: redundant empty line)

File: src/main/java/core/basesyntax/FigureSupplier.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Random randomObjectInitialization = new Random();
+    public ColorSupplier colorSupplierObjectInitialization = new ColorSupplier();
+    public final String[] figures = {"circle", "square", "right triangle", "rectangle", "isosceles trapezoid"};
```
Comment (MaliukDaria) [inline src/main/java/core/basesyntax/FigureSupplier.java:8]: same, constant naming

**Review by MaliukDaria — CHANGES_REQUESTED** [review]

**Review by MaliukDaria — CHANGES_REQUESTED** [review]
And fix checkstyle, please. Travis failed

File: src/main/java/core/basesyntax/FigureSupplier.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String[] FIGURE_LIST = {"circle", "square", "right triangle",
+            "rectangle", "isosceles trapezoid"};
+    private Random random = new Random();
+    private ColorSupplier color = new ColorSupplier();
```
Comment (MaliukDaria) [inline src/main/java/core/basesyntax/FigureSupplier.java:9]:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
```

**Review by MaliukDaria — APPROVED** [review]

File: src/main/java/core/basesyntax/FigureSupplier.java (hunk 4 above)
Comment (Artem-Kushniruk) [reply src/main/java/core/basesyntax/FigureSupplier.java:9]: Thank you

**Pull request closed without merging**